io/plugins/eu.esdihumboldt.hale.io.xls/src/eu/esdihumboldt/hale/io/xls/reader/SheetSettings.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2023 wetransform GmbH
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     wetransform GmbH <http://www.wetransform.to>
+ */
+
+package eu.esdihumboldt.hale.io.xls.reader;
+
+import java.util.Optional;
+
+import javax.xml.namespace.QName;
+
+import eu.esdihumboldt.hale.common.core.io.Value;
+import eu.esdihumboldt.hale.common.core.io.ValueProperties;
+
+/**
+ * Settings for reading a specific sheet in an Excel table.
+ * 
+ * @author Simon Templer
+ */
+public class SheetSettings {
+
+	private final String identifiedByName;
+	private final Integer identifiedByIndex;
+
+	private QName typeName;
+	private Boolean skipSheet;
+	private Integer skipLines;
+
+	/**
+	 * Create sheet settings
+	 * 
+	 * @param identifiedByName optional sheet name to identify a sheet
+	 * @param identifiedByIndex optional sheet index to identify a sheet
+	 */
+	public SheetSettings(String identifiedByName, Integer identifiedByIndex) {
+		super();
```

Comment:
There is no parent class. Do we need to call `super()` here as well?

Reply:
Eclipse added it when generating the constructor. I guess there is also not really a reason to remove it.